_config.yml

```diff
@@ -1,4 +1,4 @@
 name: Your New Jekyll Site
 markdown: redcarpet
-pygments: true
-baseurl: /figure
+highlighter: null
+baseurl: /figure
```

Comment:
I just wanted to remove the pygments highlighter since I'm not using it and it's giving a silent 404 for the pygments.css file. I can remove it for now, but you're saying there's no way to resolve conflicts in _config.yml?